spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -81,6 +97,22 @@
         expect(Stripe::Transfer).not_to(have_received(:create))
       end
     end
+
+    context "when Square notifications are not enabled" do
+      it "does not attempt to transfer the order to seller's Square dashboard" do
+        call
+        expect(square_client).not_to(have_received(:orders))
+      end
+    end
+
+    context "when Square notifications are enabled" do
+      let(:order) { order_for_square }
```

Comment:
If you use `let(:marketplace) { ... }` you can get rid of the `order_for_square` bits entirely.